official/docs/node/addresses/create_addresss.js

```diff
@@ -0,0 +1,15 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+const address = new api.Address({
+    street1: '417 MONTGOMERY ST',
+    street2: 'FLOOR 5',
+    city: 'SAN FRANCISCO',
+    state: 'CA',
+    zip: '94104',
+    country: 'US',
+    company: 'EasyPost',
+    phone: '415-123-4567',
+  });
+  
+  console.log(address);
```

Comment:
For all of these examples, you'll need to use `then` to chain together the logging to console (see our current docs for an example), this is required based on how the Node library is setup as we use async/await here. Basically "do this thing, then when it's done do the next thing", if we try console logging without the `then`, the API call may not be done and there won't be anything to log yet. This is true of every example.

```suggestion
const address = new api.Address({
    street1: '417 MONTGOMERY ST',
    street2: 'FLOOR 5',
    city: 'SAN FRANCISCO',
    state: 'CA',
    zip: '94104',
    country: 'US',
    company: 'EasyPost',
    phone: '415-123-4567',
  }).then(console.log);
```